server/plugin/subscriptions.go

```diff
@@ -203,6 +231,42 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 		Flags:      flags,
 	}
 
+	subscribedProjects := sub.ProjectTitles()
+	if len(subscribedProjects) > 0 {
+		userInfo, apiErr := p.getGitHubUserInfo(userID)
+		if apiErr != nil {
+			return errors.Errorf("Failed to get GitHub user info for userID:%s", userID)
```

Comment:
```suggestion
			return errors.Wrapf(apiErr, "failed to get GitHub user info for userID:%s", userID)
```

I see we're limiting returning errors elsewhere in this function too, though we are logging them sometimes. Is the goal to avoid revealing information about the errors to the user?